mountpoint-s3/tests/reftests/reference.rs

```diff
@@ -62,7 +62,7 @@ impl Node {
 #[derive(Debug)]
 pub struct Reference {
     /// Contents of our S3 bucket
-    remote_keys: Vec<(String, MockObject)>,
+    remote_keys: BTreeMap<String, MockObject>,
```

Comment:
This change is just to de-duplicate keys. A `HashMap` would work too, but `BTreeMap`'s ordering is deterministic.